### ISSUE TITLE
Proyecto Final 

### ISSUE THREAD

**Issue by hhgn32** [software-avanzado/project-one-requirements#6]
Que tal compañeros, cual seria la posibilidad de juntarnos durante la semana para conectarnos, yo puedo toda la semana, de las 11am hasta las 6pm, menos el miércoles, para no dejar la conexión de viernes para sábado, yo soy el coordinador del grupo 4 mi correo es hhgn32@gmail.com para que nos pongamos de acuerdo saludos.

**Comment by reimerghost** [comment]
Estoy de acuerdo, es necesario que nos juntemos para poder implementarlo entre todos. Podemos utilizar el salón 014 para coordinarnos.
